src/Pohoda/Agenda.php

```diff
@@ -189,7 +189,21 @@ protected function _addRefElement(\SimpleXMLElement $xml, string $name, $value,
         }
 
         foreach ($value as $key => $value) {
-            $node->addChild('typ:' . $key, $this->_sanitize($value), $this->_namespace('typ'));
+            if (\is_array($value)) {
+                if (array_is_list($value)) {
+                    foreach ($value as $value) {
+                        $node->addChild($namespace . ':' . $key, $this->_sanitize($value), $this->_namespace($namespace));
```

Comment:
toto je dost na mieru toho filtra na to, ze je to umiestnene v Agende. ja by som pravdupovediac idealne tuto cast kodu vobec nechytal

Reply:
pozri ako je urobeny napr. https://github.com/riesenia/pohoda/blob/master/src/Pohoda/PrintRequest/PrinterSettings.php